Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -21,40 +46,75 @@ public MarkdownTextBlock()
-            RegisterPropertyChangedCallback(FontFamilyProperty, OnPropertyChanged);
-            RegisterPropertyChangedCallback(FontSizeProperty, OnPropertyChanged);
-            RegisterPropertyChangedCallback(FontStretchProperty, OnPropertyChanged);
-            RegisterPropertyChangedCallback(FontStyleProperty, OnPropertyChanged);
-            RegisterPropertyChangedCallback(FontWeightProperty, OnPropertyChanged);
-            RegisterPropertyChangedCallback(ForegroundProperty, OnPropertyChanged);
-            RegisterPropertyChangedCallback(PaddingProperty, OnPropertyChanged);
-            RegisterPropertyChangedCallback(RequestedThemeProperty, OnPropertyChanged);
-            RegisterPropertyChangedCallback(SchemeListProperty, OnPropertyChanged);
+            // Set our style.
+            DefaultStyleKey = typeof(MarkdownTextBlock);
+            Loaded += OnLoaded;
+            Unloaded += OnUnloaded;
         }
 
         private void ThemeListener_ThemeChanged(Helpers.ThemeListener sender)
         {
             RenderMarkdown();
         }
 
+        private void OnLoaded(object sender, RoutedEventArgs e)
+        {
+            // ThemeListener Initializer: Returns a boolean which indicates if the themeListener was previously null.
+            RegisterThemeChangedHandler();
+
+            // Register for property callbacks that are owned by our parent class.
+            FontSizePropertyToken = RegisterPropertyChangedCallback(FontSizeProperty, OnPropertyChanged);
+            BackgroundPropertyToken = RegisterPropertyChangedCallback(BackgroundProperty, OnPropertyChanged);
+            BorderBrushPropertyToken = RegisterPropertyChangedCallback(BorderBrushProperty, OnPropertyChanged);
+            BorderThicknessPropertyToken = RegisterPropertyChangedCallback(BorderThicknessProperty, OnPropertyChanged);
+            CharacterSpacingPropertyToken = RegisterPropertyChangedCallback(CharacterSpacingProperty, OnPropertyChanged);
+            FontFamilyPropertyToken = RegisterPropertyChangedCallback(FontFamilyProperty, OnPropertyChanged);
+            FontStretchPropertyToken = RegisterPropertyChangedCallback(FontStretchProperty, OnPropertyChanged);
+            FontStylePropertyToken = RegisterPropertyChangedCallback(FontStyleProperty, OnPropertyChanged);
+            FontWeightPropertyToken = RegisterPropertyChangedCallback(FontWeightProperty, OnPropertyChanged);
+            ForegroundPropertyToken = RegisterPropertyChangedCallback(ForegroundProperty, OnPropertyChanged);
+            PaddingPropertyToken = RegisterPropertyChangedCallback(PaddingProperty, OnPropertyChanged);
+            RequestedThemePropertyToken = RegisterPropertyChangedCallback(RequestedThemeProperty, OnPropertyChanged);
+        }
+
+        private void OnUnloaded(object sender, RoutedEventArgs e)
+        {
+            UnhookListeners();
+            themeListener.ThemeChanged -= ThemeListener_ThemeChanged;
```

Comment:
There are a couple of peculiar "bugs" in the XAML framework right now that can lead to Unloaded being called before / without a preceeding Loaded event, so it is worth checking here if themeListener is already null before doing all this..